### PR TITLE
Add context

### PR DESCRIPTION
**Pull request by uaArsen**
Add context to document_list.html, now dynamically load documents with categories from database.

### CODE REVIEW

File: moz/main/services.py [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+
 def is_user_admin(user):
     if user and user.is_authenticated and user.is_admin:
         return True
     return False
+
+
+def get_categories_with_documents():
+    categories = Category.select().order_by(Category.title.asc())
+    documents = (MOZDocument.select().order_by(MOZDocument.title.asc()))
```
Comment (ochzhen) [inline moz/main/services.py:14]: Why do we need additional parentheses?

File: moz/main/views.py [hunk 2]
```diff
@@ -1,19 +1,26 @@
-from flask import render_template, Blueprint, current_app
+import os
+
+from flask import render_template, Blueprint, current_app, send_from_directory
 from flask_login import login_required
 
+from config import MEDIA_URL, MEDIA_ROOT
```
Comment (ochzhen) [inline moz/main/views.py]: Why not to use current_app.config["KEY"] to reference config file only once when importing config in __init__?

File: moz/templates/documents_list.html [hunk 3]
```diff
@@ -1,56 +1,34 @@
...
-            </div>
+    <div class="row marketing">
+        <div class="panel-group" id="accordion">
+            {% for c in categories %}
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <h4 class="panel-title">
+                            <a data-toggle="collapse" data-parent="#accordion" href="#collapse{{ c.id }}"> {{ c.title }}
+                                <span
+                                        class="badge pull-right">{{ c.documents|length }}</span></a>
```
Comment (ochzhen) [inline moz/templates/documents_list.html:11]: I'm not sure, but what does the sign "|" mean here?

Comment (uaArsen) [reply moz/templates/documents_list.html:11]: Its like calling document.length

File: moz/main/views.py (hunk 2 above)
Comment (uaArsen) [reply moz/main/views.py]: cant use it inside @main.route as its initilized before current_app context exist

**Review by ochzhen — APPROVED** [review]

**Pull request merged**